domains/transfer/Transfer.API/API/Transfer/Startup.cs

```diff
@@ -22,53 +23,26 @@ public static class Startup
 {
     public static void AddTransfer(this IServiceCollection services, IConfiguration configuration)
     {
-        services.AddOptions<ProjectOriginOptions>().BindConfiguration(ProjectOriginOptions.ProjectOrigin).ValidateDataAnnotations().ValidateOnStart();
-        services.AddOptions<OtlpOptions>().BindConfiguration(OtlpOptions.Prefix).ValidateDataAnnotations().ValidateOnStart();
+        services.AddOptions<ProjectOriginOptions>().BindConfiguration(ProjectOriginOptions.ProjectOrigin)
+            .ValidateDataAnnotations().ValidateOnStart();
+        services.AddOptions<OtlpOptions>().BindConfiguration(OtlpOptions.Prefix).ValidateDataAnnotations()
+            .ValidateOnStart();
 
-        Configuration.Setup()
-            .UseEntityFramework(ef => ef
-                .AuditTypeExplicitMapper(config => config
-                    .Map<TransferAgreement, TransferAgreementHistoryEntry>((evt, eventEntry, historyEntity) =>
```

Comment:
Why has this been moved to program.cs?

Reply:
It seems that Audit.net only can have one Configuration.Setup, but i might be able to move to mapper to the Startup - i will look into it.

Reply:
Audit.NET is now used in different sub systems